app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayFragment.kt

```diff
@@ -358,6 +360,12 @@ class TabsTrayFragment : AppCompatDialogFragment() {
 
     @VisibleForTesting
     internal fun selectTabPosition(position: Int, smoothScroll: Boolean) {
+        if (position == Page.PrivateTabs.ordinal) {
```

Comment:
This method is for selecting the tab position, so let's make that is all it does. We have other UI "features" that are based on the selected page, but if we put them all here it would be a pretty big method. 😄 

If we use a composable approach with the `AbstractBinding` like the others above, we can isolate this specific feature in one class so we can add/remove it, test it easily and without needing to mock lots of Android framework code, and most importantly for me, it's easier to follow what it's trying to do.

Have a look at the `FloatingActionButtonBinding`, which uses settings and observes the selected page from the store.